frontend/src/components/loaders/FullScreenLoader.component.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './FullScreenLoader.scss';
+
+const FullScreenLoader = ({ isVisible }: { isVisible: boolean }) =>
+  isVisible ? (
+    <div className={'full-screen-container'}>
+      <img src={require('assets/loaders/factory-loader.gif')} alt={'Loading...'} width={150} />
```

Comment:
Pourquoi ne pas importer le gif en haut du fichier plutôt ?

Reply:
Sais pas, j'ai vu un exemple comme ça alors je l'ai reproduit, ça change quelque chose ? De toute façon l'image est suffisamment petite pour que le chargement soit négligeable

Reply:
C'était plutôt par souci d'homogénéité